packages/login/src/containers/LoginFormContainer.js

```diff
@@ -0,0 +1,24 @@
+import {connect} from 'react-redux'
+import {injectIntl} from 'react-intl'
+
+import LoginForm from '../components/LoginForm'
+import {changePage, setUsername, setPassword} from '../modules/login/actions'
+import {login} from '../modules/actions'
+
+const mapActionCreators = {
+  changePage,
+  setUsername,
+  setPassword,
+  login
+}
+
+const mapStateToProps = state => {
```

Comment:
can be simplified as in LoginContainer:

const mapStateToProps = state => ({
   ...
})

--> other containers too.